src/App.tsx

```diff
@@ -37,9 +37,11 @@ const App = () => {
 
 	const tenantId = urlParams.get('tenant') || process.env.DESCOPE_TENANT_ID;
 
+	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
+
 	return (
 		<AuthProvider projectId={projectId} baseUrl={baseUrl}>
-			<div className="app">
+			<div className="app" style={{ backgroundColor }}>
```

Comment:
I have consider to do something clever to support getting hexa color without the `#` (because pasing `#` in url encoding is cumbersome - `%23A2A2A2` for example..) 

but eventually decided not to because the solution may conflict with colors (though it can be solved with some regex like `/^[0-9A-Fa-f]{6}$/`)